ansible-runner/context/app/project/roles/mirror_appliance/templates/postgres.service.j2

```diff
@@ -10,7 +10,7 @@ TimeoutStartSec=5m
 ExecStartPre=-/bin/rm -f %t/%n-pid %t/%n-cid
 ExecStart=/usr/bin/podman run \
     --name quay-postgres \
-    -v {{ quay_root }}/pg-data:/var/lib/pgsql/data:Z \
+    -v quay-postgres-data:/var/lib/pgsql/data:Z \
```

Comment:
Can we still keep the named volume somewhere inside `quay_root`? I think we should keep the ability that a user can specify their pg mount path especially since it can get quite large over time.

Reply:
I was running into permissions errors as a non-root user when using bind mounts with the postgres container. The postgres user is different than the non-root user and I was having trouble writing to `quay_root`. Tried a couple different things, but couldn't get it working. Not sure if it's an selinux thing, but dropping in a named volume worked so I went with it. Maybe you know how I can work around user permissions?

Reply:
SELinux forces you to set special permissions on a directory before being able to mount it. That is the reason for these steps in each of the service setup tasks. https://github.com/quay/mirror-registry/blob/main/ansible-runner/context/app/project/roles/mirror_appliance/tasks/install-postgres-service.yaml#L7 Maybe we have to update the user there?